install_config/advanced_ldap_configuration/index.adoc

```diff
@@ -0,0 +1,20 @@
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+{product-title} Advanced Lightweight Directory Access Protocol (LDAP)
+Configuration covers the following topics:
+
+- link:../advanced_ldap_configuration/sssd_for_ldap_failover.html[Setting up SSSD for LDAP Failover]
+
+- link:../advanced_ldap_configuration/configuring_form_based_authentication.html[Configuring Form-Based Authentication]
+
+- link:../advanced_ldap_configuration/configuring_extended_ldap_attributes.html[Configuring Extended LDAP Attributes]
+
+{product-title} can sync LDAP records with internal {product-title} records,
```

Comment:
Syncing the LDAP records with internal records isn't going to work with the SSSD integration as described here. I haven't looked into whether or how this can be done. That's probably worth another investigation and blog post, but I don't know if I'll have time in the near future to do it.